modules/viz/include/opencv2/viz/viz3d.hpp

```diff
@@ -142,6 +142,12 @@ namespace cv
             */
             Camera getCamera() const;
 
+            /** @brief Sets pose the camera roll. This will spin the camera about its axis.
+
+            @param angle The new roll angle in degrees of the camera.
+             */
+            void cameraRoll(double angle);
```

Comment:
```
void vtkCamera::Roll(double angle)
{
  double newViewUp[3];
  this->Transform->Identity();

  // rotate ViewUp about the Direction of Projection
  this->Transform->RotateWXYZ(angle,this->DirectionOfProjection);

  // okay, okay, TransformPoint shouldn't be used on vectors -- but
  // the transform is rotation with no translation so this works fine.
  this->Transform->TransformPoint(this->ViewUp,newViewUp);
  this->SetViewUp(newViewUp);
}
```
The Roll method of vtkCamera rotates the camera around the view plane normal. If I compute and set the camera position:
```
cv::Affine3f cam_pose = cv::viz::makeCameraPose( currentPoint, cv::Point3d( 0, 0, 0 ), cv::Point3d( 0, -1, 0 ) );
myWindow_.setViewerPose( cam_pose );
```
After this positioning how can I do the in-plane rotation of the camera? If I do as you write in your comment I lose the previous rotations on the other axes. 
Vtkcamera stores the DirectionOfProjection in order to compute the new up vector of the camera, but out of the viz module we don't have this information.